### ISSUE TITLE
Nonetype Object in Faceboxes

### ISSUE THREAD

**Issue by MarioJayakumar** [samj1912/fbrecog#8]
Line 83 in init.py, faceboxes is sometimes initialized to NoneType, which causes an error when trying to iterate through it?

**Comment by nickvandewiele** [comment]
I agree that the case `None` for `faceboxes` is not properly caught there. 

But, in my experience, when None is returned, it is a symptom of another problem upstream, e.g. a failed authentication to the FB api. Check if renewing the cookie, or `fb_dtsg` parameter help in avoiding `None`.
